staging/src/k8s.io/apiserver/pkg/server/config_test.go

```diff
@@ -98,6 +98,7 @@ func TestNewWithDelegate(t *testing.T) {
   "paths": [
     "/apis",
     "/bar",
+    "/configz",
```

Comment:
is this endpoint protected by default? If not, isn't the admission chain security sensitive?

Reply:
>is this endpoint protected by default?

yes